packages/canner-helpers/src/createFakeData.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+/**
+ * producing the immutable data with the given argument
+ *
+ * if object => recurse its children
+ * if array => new List
+ * if string => ''
+ * if boolean => false
+ * if number => 0
+ * if toOne relation => null
+ * if toMany relation => null
+ * if dateTime => string
+ */
+
+import invariant from 'invariant';
+import faker from 'faker';
+import {mapSchema} from './utils';
+import type {Schema, SchemaMap} from './types';
+// createFakeData = (schema: Object, listLength: number) => *
+
+
+export default function createFakeData(schema: Schema | SchemaMap, listLength: number = 0) {
+  if (!schema) {
+    return null;
+  }
+  if ('type' in schema) {
+    return loop(((schema: any): Schema), listLength);
+  }
+  return mapSchema(((schema: any): SchemaMap), (schema: Schema) => loop(schema, listLength));
+}
+
+
+function loop(schema: Schema, listLength: number) {
+  let result: any;
+
+  switch (schema.type) {
+    case 'object':
+      result = mapSchema(schema.items, loop);
+      result.__typename = result.__typename || null;
+      break;
+    case 'array':
+      result = [];
+      for (let i = 0; i < listLength; ++i) {
+        result.push(mapSchema(schema.items.items, loop));
```

Comment:
@chihsuan, should consider other cases here such as:
- Array of string: `{type: 'array', items: {type: 'string'}}`,   
- Array of number: `{type: 'array', items: {type: 'number'}}`,